scripts/trace_flowcell.py

```diff
@@ -0,0 +1,152 @@
+        folders = {}
+        states = {}
+        choice = 0
+        hitcount = 0
+        outstring = "\n\tstatus\trunfolder_link\n"
+        for runfolder in all_runfolders:
+            link = runfolder["link"]
+            state = runfolder["state"]
+            if search_term in link and not "rchive" in link and not "biotank" in link:
+                hitcount += 1
+                dirs = link.split('/')
+                folders[hitcount] = dirs[-1]
+                states[hitcount] = state
+                outstring += "{}\t{}\t{}\n".format(hitcount, state, folders[hitcount])
+        
+        if hitcount < 1:
+            raise ValueError("No hit")
+        if hitcount > 1:
+            print outstring
+            choice = int(raw_input("Which runfolder to trace: "))
+        else:
+            choice = 1
+        return folders[choice], states[choice]
+
+def get_executions(actions):
+    for a in actions:
+        bashCommand = "st2 execution get {}".format(a["id"])
+        process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE) 
+        return process.communicate()
+
+        
+def print_stackstorm_id(sorted_actions):
+    for a in sorted_actions:
+        print a["id"]
+        
+
+if __name__ == "__main__":
+
+    # Try to load access token from environment - fall back if not available
+    try:
+        access_token = os.environ['ST2_AUTH_TOKEN']
+    except KeyError:
+            print("Could not find ST2_AUTH_TOKEN in environment. "
+                "Please set it to your st2 authentication token.")
+    access_headers = {"X-Auth-Token": access_token}
+        
+    # parse the input
+    parser = argparse.ArgumentParser(description="Gets execution ids associated with a flowcell "
+                                     "(e.g. a runfolder) and a workflow. It is used to trace the flowcell"
+                                     "--flowcell 000000000-ABGT6_testbio14 ")
+    parser.add_argument('--flowcell', required=True, help='Specify the flowcell you want to trace. E.g. --flowcell HTTT3CCXY')
+    parser.add_argument('--noverify', action="store_false")
+    parser.add_argument('--api_base_url', default="http://localhost:9101/v1")
+    parser.add_argument('--config', default="/opt/stackstorm/packs/snpseq_packs/config.yaml")
+    parser.add_argument('--workflow', default="workflow")
+    args = parser.parse_args()
+
+    # find the runfolder to trace
+    runfolder = PickRunfolder(args.config)
+    try:
+        folder2trace, folder_state = runfolder.choose_runfolder(args.flowcell)
+    except ValueError: 
+        print "\n\tNo hits found, will terminate.\n"
+        sys.exit()
+    print "Will trace {}.".format(folder2trace)
+
+    # trace the runfolder
+    myTraces = GetTraces(args.api_base_url, access_headers, args.noverify, args.workflow)
+    sorted_actions = myTraces.get_sorted_actions_from_tag(folder2trace)
+    
+    # output the result
+    output, error = get_executions(sorted_actions)
```

Comment:
see comment above regarding `get_executions`

Reply:
will print executions directly in function